R/chromote.R

```diff
@@ -1,7 +1,9 @@
 #' Chromote class
 #'
+#' @description
 #' This class represents the browser as a whole.
 #'
+#' @details
```

Comment:
IMO you could remove line 3 and make lines 7-10 the description. (Or even the whole text block could arguably be the description)